lib/internal/modules/esm/load.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { defaultGetFormat } = require('internal/modules/esm/get_format');
+const { defaultGetSource } = require('internal/modules/esm/get_source');
+const { translators } = require('internal/modules/esm/translators');
+
+async function defaultLoad(url, context) {
+  let {
+    format,
+    source,
+  } = context;
+
+  if (!translators.has(format)) format = defaultGetFormat(url);
+
+  if (
+    format === 'builtin' ||
+    format === 'commonjs'
+  ) {
+    source = null;
+  } else if (source == null) {
+    source = await defaultGetSource(url, { format });
+  }
+
+  return {
+    format,
+    source,
```

Comment:
Did you run `make lint` on your branch? Trailing commas are disallowed. (Nowadays most devs prefer them, but no one wants to flip the linter to require them and then there’s a huge PR to add them everywhere, and all open PRs get merge conflicts—basically changing at this point is more trouble than it’s worth).
```suggestion
    source
```

Reply:
All tests and linters pass on the latest commit. Trailing commas are allowed.

Reply:
Yep: it passes locally and in CI :)